### ISSUE TITLE
[BUG] should ignore uppercase lowercase

### ISSUE THREAD

**Issue by renyiwei-xinyi** [microsoft/playwright-java#401]
```
package com.microsoft.playwright.options;

public enum SameSiteAttribute {
    STRICT,
    LAX,
    NONE;

    private SameSiteAttribute() {
    }
}
```

![image](https://user-images.githubusercontent.com/55709533/115098612-8cef3e00-9f63-11eb-9ad0-3be6cc42b896.png)

**Comment by yury-s** [comment]
Which Playwright version are you using? I've added a test which passes in 1.10.0 and in 1.11.0-SNAPSHOT, you may need to update to a newer version.

**Comment by renyiwei-xinyi** [comment]
1.10.0  and 1.11.0-SNAPSHOT

**Comment by renyiwei-xinyi** [comment]
this Java Enum SameSiteAttribute is STRICT /  LAX / NONE ,  but  js expected one of (Strict|Lax|None)

**Comment by renyiwei-xinyi** [comment]
![image](https://user-images.githubusercontent.com/55709533/118113260-a5def800-b418-11eb-93a1-e9c6e0ba1dad.png)

**Comment by renyiwei-xinyi** [comment]
`@JsonFileSource(files = "/example/cookies2.json")
    void test_12839(List<Cookie> cookies){
        System.out.println(JSONUtil.toJsonPrettyStr(cookies));

        Browser browser = Playwright.create().chromium().launch();

        BrowserContext context = browser.newContext();
        // test add cookies
        context.addCookies(cookies);
    }`

**Comment by renyiwei-xinyi** [comment]
```
@JsonFileSource(files = "/example/cookies2.json")
    void test_12839(List<Cookie> cookies){
        System.out.println(JSONUtil.toJsonPrettyStr(cookies));

        Browser browser = Playwright.create().chromium().launch();

        BrowserContext context = browser.newContext();
        // test add cookies
        context.addCookies(cookies);
    }
```

**Comment by renyiwei-xinyi** [comment]
cookies2.json
```
[
  {
    "name": "1P_JAR",
    "value": "2021-05-13-10",
    "url": null,
    "domain": ".google.com",
    "path": "/",
    "expires": 1.623493118655839E9,
    "httpOnly": false,
    "secure": true,
    "sameSite": "NONE"
  }
]
```

**Comment by yury-s** [comment]
Try with the latest release (v1.11.0), it already contains the fix.  v1.10.0 doesn't have the fix and your 1.11.0-SNAPSHOT is likely too old (note that -SNAPSHOT releases are updated without changing the version in maven).

**Comment by renyiwei-xinyi** [comment]
ok, thinks.  i have found BUG
the type change is fallacious
